### PR TITLE
v2.x.x split packages 

### PR DESCRIPTION
**Pull request by wasabeef**
### What does this change?

Split for build_runner and command-lines.

**For Command-line**
  - dir:`packages/command`
  - pub.dev: `flutter_gen`

Just as usual.
```sh
$ brew install FlutterGen/tap/fluttergen
$ dart pub global activate flutter_gen
```

**For Build Runner**
- dir: `packages/runner`
- pub.dev: `flutter_gen_runner`

New package.
```yaml
dev_dependencies:
  build_runner:
  flutter_gen_runner: ^2.0.0
```

**Common Package (Core)**
- dir: `packages/core`
- pub.dev: `flutter_gen_core` *unlisted.

### What is the value of this and can you measure success?

Pass https://github.com/FlutterGen/flutter_gen/issues/53

### CODE REVIEW

File: packages/command/pubspec.yaml [hunk 1]
```diff
@@ -16,19 +17,8 @@ executables:
...
-  args: ^1.6.0
-  path: ^1.7.0
-  yaml: ^2.2.1
-  mime: ^0.9.7
-  xml: ^4.2.0
-  dart_style: ^1.3.6
-  dartx: ^0.5.0
-  color: ^2.1.1
-  merge_map: ^1.0.2
+  flutter_gen_core: ^0.0.2
```
Comment (wasabeef) [inline packages/command/pubspec.yaml:20]: After the review is complete, I'll release 2.0.0 and change it.

File: _internal/build.yaml [hunk 2]
```diff
@@ -1,17 +0,0 @@
-targets:
```
Comment (lcdsmao) [inline _internal/build.yaml:1]: We should move this file into `/packages/core`

**Review by lcdsmao — COMMENTED** [review]
We should modify comment in `packages/core/lib/settings/pubspec.dart`

File: _internal/build.yaml (hunk 2 above)
Comment (wasabeef) [reply _internal/build.yaml]: I understood completely.

**Review by lcdsmao — APPROVED** [review]
🎉

**Pull request merged**